exporters/tests/test_deserialization.py

```diff
@@ -10,8 +11,12 @@
     MissingFieldError,
     deserialize,
     nested,
+    retain_source,
 )
 
+# TODO: test with kubernetes.dynamic.resource.ResourceField
+# TODO: test retaining source
+
```

Comment:
Actually, I should rebase because this was handled in the other branch, but not removed here.